pootle/apps/pootle_app/management/commands/project_serializers.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import CommandError
+from django.utils.translation import ugettext_lazy as _
+
+from pootle.core.delegate import serializers, deserializers
+from pootle_app.management.commands import PootleCommand
+from pootle_project.models import Project
+from pootle_store.models import Store
+
+
+class Command(PootleCommand):
+    help = "Update database stores from files."
+    process_disabled_projects = True
+
+    def add_arguments(self, parser):
+        super(Command, self).add_arguments(parser)
+        parser.add_argument(
+            'project',
+            type=str,
+            default="",
+            nargs="?",
+            help='Pootle project')
+        parser.add_argument(
+            '--list',
+            action='store_true',
+            dest='list',
+            help="List available de/serializers")
+        parser.add_argument(
+            '--set-serializers',
+            action='store',
```

Comment:
i considered this but wouldnt this mean you would have to type somethin like

```
pootle project_serializers myproject --set-serializer foo --set-serializer bar
```

also, because the order is very much significant it feels better just using comma separated arg